### PR TITLE
Fix https://github.com/NuGet/Home/issues/1157

### PR DESCRIPTION
**Pull request by yishaigalatzer**

### CODE REVIEW

File: test/NuGet.Commands.Test/RestoreCommandTests.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.Diagnostics;
```
Comment (yishaigalatzer) [inline test/NuGet.Commands.Test/RestoreCommandTests.cs]: will remove

File: test/NuGet.Commands.Test/RestoreCommandTests.cs [hunk 2]
```diff
@@ -130,6 +131,70 @@ public class RestoreCommandTests : IDisposable
...
+                        ""warn"": false
+                    }
+                }
+            }");
+
+            var specPath = Path.Combine(projectDir, "TestProject", "project.json");
+            var spec = JsonPackageSpecReader.GetPackageSpec(configJson.ToString(), "TestProject", specPath);
+
+            //var request = new RestoreRequest(spec, sources, packagesDir);
+            //request.LockFilePath = Path.Combine(projectDir, "project.lock.json");
```
Comment (emgarten) [inline test/NuGet.Commands.Test/RestoreCommandTests.cs:160]: clean up

**Comment by emgarten** [comment]
:rocket:

**Comment by deepakaravindr** [comment]
:shipit:

**Comment by yishaigalatzer** [comment]
Merged through https://github.com/NuGet/NuGet3/commit/00053876b86fd29b46bfdaccb76dfd5ceba7701b

**Pull request closed without merging**